datasets/cn/national_congress/crawler.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+import csv
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def crawl_item(input_dict: dict, context: Context):
+    name = input_dict.pop("姓名/Name")
+    ethnicity = input_dict.pop("民族/Ethnicity")
+    gender = input_dict.pop("性别/Gender")
+    delegation = input_dict.pop("代表团/Delegation")
+
+    entity = context.make("Person")
+    entity.id = context.make_slug(name, ethnicity, gender, delegation)
+
+    entity.add("name", name, lang="chi")
+    entity.add("gender", context.lookup("gender", gender))
+    entity.add("ethnicity", ethnicity, lang="chi")
+
+    position = h.make_position(
+        context, "Member of the National People’s Congress", country="cn"
+    )
+    categorisation = categorise(context, position, is_pep=True)
+
+    occupancy = h.make_occupancy(
+        context,
+        entity,
+        position,
+        True,
+        categorisation=categorisation,
+    )
+
+    position.add("subnationalArea", delegation, lang="chi")
```

Comment:
actually subnationalArea is for when the jurisdiction of the position is limited to a subnational area. In this case their jurisdiction is national, they just represent a subnational region of their delegation, so let's skip this property.